macinstall/kubescape.rb

```diff
@@ -0,0 +1,41 @@
+class Kubescape < Formula
```

Comment:
Kubescape can be downloaded as [follows](https://github.com/kubescape/kubescape#install-on-macos).

Reply:
yeah but how do I include those commands for mac updation?